maven.htmlui/nbproject/project.xml

```diff
@@ -57,6 +57,23 @@
                     <code-name-base>org.netbeans.html.ko4j</code-name-base>
                     <run-dependency/>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.libs.junit4</code-name-base>
```

Comment:
This is probably wrong: it should be a test only dependency. Not dependency of regular code.